<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCMatrixOutputNodeWidget</class>
 <widget class="QWidget" name="BCMatrixOutputNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="m_pNodeWidget1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="m_pNodeIconLabel1">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/matrix/output.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_pStatusLabel1">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_pNodeWidget2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="m_pNodeIconLabel2">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/matrix/output.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_pStatusLabel2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
